olusoladavid/my-diary-react-redux
# #161775683 Implement signup page

#### What does this PR do?

- Implement signup page

#### Description of tasks to be completed

- [x] - Add signup page
- [x] - Implement validation and fetch functionality

#### How should this be manually tested?

- Pull branch to local machine
- `npm start`
- Navigate to '/signup'

- The signup form should be working with validation and submission in place
#### What are the relevant pivotal tracker stories?

[#161775683](https://www.pivotaltracker.com/story/show/161775683)
